src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java

```diff
@@ -244,7 +245,14 @@ public void setSources(
                 return;
             }
 
-            endpoint.addReceiveSsrc(source.getSSRC());
+            if (audioSources.contains(source))
```

Comment:
i considered using the original audio and video lists, but since there's some filtering that didn't seem great (and would also result in some duplicated code) so did it this way.  open to another idea though.

Reply:
I think this is fine.